packages/build-config/shims/worker-shim.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+require('babel-polyfill');
+
+(function execute() {
+  var entryPoint = require('hops-worker-entry-point');
+  if (entryPoint.__esModule) {
+    entryPoint = entryPoint.default;
+  }
+  entryPoint(HOPS_ASSETS); // eslint-disable-line no-undef
+  if (module.hot) {
```

Comment:
Not sure how Service Workers behave with hot module reloading.. needs more investigation.

Reply:
Shall we disable it for now?